lib/apiservers/engine/network/utils.go

```diff
@@ -564,7 +564,7 @@ func PortForwardingInformation(vc *viccontainer.VicContainer, ips []string) []ty
 	//c := cache.ContainerCache().GetContainer(cid)
```

Comment:
I know this isn't your code but can we delete these commented-out lines?

Reply:
nope ;)